deployment/playbooks/includes/defaults.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Set default values
+  set_fact:
+    volttron_home: "{{ volttron_home | default('{{ ansible_env.HOME }}/.volttron') }}"
+    volttron_root: "{{ volttron_root | default('{{ ansible_env.HOME }}/volttron') }}"
+    host_config: "{{ host_config | default('{{ ansible_env.HOME }}/host.config.yml') }}"
+    host_configs_dir: "{{ host_configs_dir | default('{{ ansible_env.HOME }}/configs') }}"
+    enable_web: no
```

Comment:
Can we follow same format as above lines. This is a low priority task that Ben can take up later